srcpkgs/Clipboard/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'Clipboard'
+pkgname=Clipboard
+version=0.2.1r2
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libX11-devel wayland-devel"
+short_desc="Cut, copy, and paste anything, anywhere, all from the terminal"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Slackadays/Clipboard"
+distfiles="https://github.com/Slackadays/Clipboard/archive/refs/tags/${version}.tar.gz"
+checksum=3d88f14aa38530f0af74c492e0e708b206b56e67bc6d04275dfe12e6983c865a
+
+if [ "${XBPS_TARGET_NO_ATOMIC8}" ]; then
+	makedepends+=" libatomic-devel"
+	CXXFLAGS+=" -latomic"
+fi
+
+# X11 support won't build on these targets
+case "${XBPS_TARGET_MACHINE}" in
+	i686|armv6*|armv7*) configure_args="-DNO_X11=1" ;;
```

Comment:
```suggestion
	i686*|armv6*|armv7*) configure_args="-DNO_X11=1" ;;
```
If it's because they are 32 bits, this should use XBPS_TARGET_WORDSIZE

Reply:
You're right, adjusted for that